internal/plugin/connectors/tcp/mssql/connection_details_test.go

```diff
@@ -125,28 +132,39 @@ func TestConnectionDetails_NewSSLOptions(t *testing.T) {
 			args: args{
 				credentials: map[string][]byte{},
 			},
-			expected: map[string]string{},
+			expected: sslModeToBaseParams[sslModeRequire],
+		},
+		{
+			description: "sslmode: disable",
```

Comment:
a thought: I know this concern hasn't been introduced by this particular PR but I'm thinking we should be descriptive of our test cases. Going to look into some best practices for this.

Reply:
Agreed, I went through and tried to make them more descriptive, but I'm interested in learning the best practice for this as well. We should bring the team together for a small chat on that topic and document something.